examples/conf-quickstart/druid/broker/runtime.properties

```diff
@@ -3,7 +3,7 @@ druid.port=8082
 
 # HTTP server threads
 druid.broker.http.numConnections=5
-druid.server.http.numThreads=8
```

Comment:
the exception seems to be related to tranquility, not sure if this is needed here as well ?